location/src/androidLibMain/AndroidManifest.xml

```diff
@@ -14,7 +14,7 @@
                 android:name=".EnableLocationActivity"
                 android:theme="@style/Location"/>
 
-        <receiver android:name=".GoogleLocationUpdatesBroadcastReceiver"  android:exported="true">
+        <receiver android:name=".GoogleLocationUpdatesBroadcastReceiver" android:exported="true">
```

Comment:
From [source](https://developer.android.com/about/versions/12/behavior-changes-12#exported) as far as I understand it doesn't require to be `true` in all cases, but mainly when  the `Launcher` is specified.
>If the app component includes the LAUNCHER category, set android:exported to true. In most other cases, set android:exported to false.